multiversion/src/main/scala/multiversion/diagnostics/ConflictingTransitiveDependencyDiagnostic.scala

```diff
@@ -26,11 +26,17 @@ class ConflictingTransitiveDependencyDiagnostic(
     def pretty(xs: Iterable[Any]): String =
       if (xs.isEmpty) ""
       else xs.mkString(" ", ", ", "")
+    val depsStr =
+      (foundVersions
+        .map { v =>
+          s"'dependency = ${Dependency(module, v).repr}'"
```

Comment:
```suggestion
          s"'dependency: ${Dependency(module, v).repr}'"
```

Reply:
This error message is suggesting BUILD file change during export, so `=` should be ok here.

Reply:
Oh, but you're right about "export" command using with YAML as input too. Maybe we should somehow change the error message format if it came from "pants-export".